tests/spread/core24/craftgit/task.yaml

```diff
@@ -0,0 +1,8 @@
+summary: Pack a snap that uses craft.git
+
+restore: |
+  snapcraft clean
+  rm -f ./*.snap
+
+execute: |
```

Comment:
Do you mean to run in this in destructive mode, with lxd, or does it not matter?

I'm double-checking because snapcraft's spread tests have the sneaky behavior of setting `SNAPCRAFT_BUILD_ENVIRONMENT=host` by default.

Reply:
It shouldn't matter, but from what I saw `core24` tests run in managed mode:
https://github.com/canonical/snapcraft/blob/cf708937e54eaba225ba6e84b77c176ae0084a23/spread.yaml#L239-L244

Reply:
Oh nice, I forgot about that.